### PR TITLE
feat: cookie constent changes

### PR DESCRIPTION
**Pull request by ArshdeepGrover**
Cookie consent changes

image

### CODE REVIEW

File: projects/commudle-admin/src/app/app.component.ts [hunk 1]
```diff
@@ -58,16 +58,6 @@ export class AppComponent implements OnInit, AfterViewChecked, OnDestroy {
...
-        this.windowService.open(CookieConsentComponent, {
-          title: "Let's Share Cookies!",
-          initialState: NbWindowState.MAXIMIZED,
-          windowClass: 'cookie-consent',
-        });
-      }, 3000);
-    }
-
     if (this.cookieConsentService.isCookieConsentAccepted()) {
       this.cookieAccepted = true;
```
Comment (arpansac) [inline projects/commudle-admin/src/app/app.component.ts:59]: Why has settimeout been removed?

File: projects/shared-components/cookie-consent/cookie-consent.component.ts [hunk 2]
```diff
@@ -1,33 +1,26 @@
...
+  constructor(private cookieConsentService: CookieConsentService, private cookieService: CookieService) {}
 
-  ngOnInit() {
-  }
+  ngOnInit() {}
 
   acceptCookieConsent() {
     this.cookieConsentService.acceptCookieConsent();
-    this.ref.close();
+    this.close.emit(true);
```
Comment (arpansac) [inline projects/shared-components/cookie-consent/cookie-consent.component.ts]: Not correct

File: projects/commudle-admin/src/app/app.component.html [hunk 3]
```diff
@@ -29,6 +29,7 @@
   </nb-layout-footer>
 </nb-layout>
 
-<app-chats-container *ngIf="currentUser && cookieAccepted"></app-chats-container>
+<app-chats-container *ngIf="currentUser"></app-chats-container>
```
Comment (arpansac) [inline projects/commudle-admin/src/app/app.component.html]: Let's get cookie accepted here also

File: projects/shared-components/cookie-consent/cookie-consent.component.ts [hunk 4]
```diff
@@ -1,33 +1,41 @@
...
   ngOnInit() {
+    if (this.isBrowser && !this.cookieConsentService.isCookieConsentAccepted()) {
+      setTimeout(() => {
+        this.cookieConstent = false;
+      }, 3000);
+    }
+
+    if (this.cookieConsentService.isCookieConsentAccepted()) {
+      this.cookieConstent = true;
+    }
```
Comment (arpansac) [inline projects/shared-components/cookie-consent/cookie-consent.component.ts]: This code can be optimized, already discussed

File: projects/shared-components/cookie-consent/cookie-consent.component.ts [hunk 5]
```diff
@@ -1,33 +1,41 @@
...
   acceptCookieConsent() {
     this.cookieConsentService.acceptCookieConsent();
-    this.ref.close();
+    this.cookieConstent = true;
   }
 
   disagreeCookieConsent() {
     this.cookieService.deleteAll();
     window.location.href = 'about:blank';
   }
```
Comment (arpansac) [inline projects/shared-components/cookie-consent/cookie-consent.component.ts]: comment this out

File: tailwind.config.js [hunk 6]
```diff
@@ -18,6 +18,9 @@ module.exports = {
         'span-22': 'span 22 / span 22',
       },
     },
+    fontFamily: {
+      'inter': 'Inter',
+    },
```
Comment (arpansac) [inline tailwind.config.js]: remove

**Review by arpansac — CHANGES_REQUESTED** [review]

File: projects/commudle-admin/src/app/app.component.ts (hunk 1 above)
Comment (ArshdeepGrover) [reply projects/commudle-admin/src/app/app.component.ts:59]: set back again!

**Review by arpansac — APPROVED** [review]

**Pull request merged**